cl_dll/drawingutility.cpp

```diff
@@ -0,0 +1,78 @@
+#include<windows.h>
+#include <GL/gl.h>
+#include <math.h>
+#include "drawingutility.h"
+
+#ifndef M_PI
```

Comment:
Don't define PI, it's already defined in math.h (on Windows you need _USE_MATH_DEFINES or something like that).